### ISSUE TITLE
Pathogen name change

### ISSUE THREAD

**Issue by jack-bischoff** [greymatter-io/pathogen-greymatter#12]
Should the repo name change from `pathogen` to a name evocative of the GM CLI since pathogen will not technically be part of greymatter, per the conversation in the product demo on 10/29?

**Comment by kaitmore** [comment]
Yes, after speaking to Chris H we definitely want to remove any references to pathogen as it's not relevant to CLI templating from an end-user perspective.

**Comment by chrisbsmith** [comment]
As any FYI, I've been going with `greymatter-templates` in other environments.
